test/issue2698-cimportpaths-broken-with-dmd-ldc/c_headers/foo.h

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+
+int bar(void)
+{
+	printf("func bar in foo.h\n");
+}
```

Comment:
Writing the implementation in .h files is a terrible practice.
How about declaring the type as a struct or something instead of a function?